src/data/contributors/AyazMirza.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Ayaz Mirza",
+    "skills": ["C++", "HTML & CSS", "SQL"],
+    "cause": "I code for acquiring a skill",
```

Comment:
I code for is already the part of the string :)

Reply:
Updated the same thanks:)